src/toil/jobStores/abstractJobStore.py

```diff
@@ -234,10 +234,11 @@ def _checkJobStoreCreation(create, exists, jobStoreString):
             raise JobStoreCreationException("The job store '%s' does not exist, so there "
                                             "is nothing to restart." % jobStoreString)
 
-    def importFile(self, srcUrl):
+    def importFile(self, srcUrl, sharedFileName=None):
         """
         Imports the file pointed at by sourceUrl into job store. The jobStoreFileId of the new
-        file is returned.
+        file is returned. If a shared file name is given the file is imported as a shared file
+        and its name is returned.
```

Comment:
Return None instead. Shared files don't have an ID (from the user's perspective) so returning the shared file name (which is also a string) may introduce bugs where a shared file name is used as a file ID. From that perspective None is safer. Besides, there really is no value in returning an argument.